src/main/java/com/coveo/spillway/limit/Limit.java

```diff
@@ -78,12 +95,36 @@ public Duration getExpiration() {
     return definition.getExpiration();
   }
 
+  public Duration getExpiration(T context) {
+    return findLimitOverride(context).map(p -> p.getExpiration()).orElse(getExpiration());
+  }
+
   public int getCapacity() {
     return definition.getCapacity();
   }
 
+  public int getCapacity(T context) {
+    return findLimitOverride(context).map(p -> p.getCapacity()).orElse(getCapacity());
+  }
+
+  public Set<LimitOverride> getLimitOverrides() {
+    return limitOverrides;
+  }
+
   @Override
   public String toString() {
     return definition.toString();
   }
+
+  private Optional<LimitOverride> findLimitOverride(T context) {
+    String property = getProperty(context);
+
+    for (LimitOverride limitOverride : limitOverrides) {
```

Comment:
Suggestion, not a requirement 😄 
```
return limitOverrides.stream().findAny(override -> override.getProperty().equals(property));
```

Change to `findFirst` if there can be more match than one and order is important.